css/index.css

```diff
@@ -1,3 +1,331 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Site-wide Styles */
+
+* {
+
+	box-sizing: border-box;
+
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+nav {
+
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px 0px;
+
+}
+
+a {
+
+	text-decoration: none;
+	color: grey;
+
+}
+
+h1 {
+
+	font-size: 27px;
+
+}
+
+p {
+
+	line-height: 1.2;
+
+}
+
+ul {
+
+	list-style: square;
+	list-style-position: inside;
+
+}
+
+footer {
+
+	text-align: center;
+	margin: 20px 0px;
+
+}
+
+.container {
+
+	width: 1016px;
+	margin: auto;
+	padding: 20px 70px 0px 70px;
+
+}
+
+/* Home Styles */
+
+.homeBanner {
+
+	display: flex;
+	border-bottom: 1px solid black;
+	margin-bottom: 40px;
+	padding: 5% 0px 10% 0px;
+	justify-content: space-between;
+
+}
+
+.header-text {
+
+	width: 50%;
+	text-align: center;
+
+}
+
+.header-text h1 {
+
+	font-size: 78px;
+
+}
+
+.startButton {
+
+	border: 1px solid black;
+	padding: 10px 50px;
+	background-color: white;
+
+}
+
+.section1 {
+
+	display: flex;
+	justify-content: space-between;
+
+}
+
+.section1 section {
+
+	width: 49%;
+
+}
+
+.middle-img {
+
+	margin: 40px 0px;
+
+}
+
+.section2 {
+
+	display: flex;
+	justify-content: space-between;
+
+}
+
+.section2 section {
+
+	width: 32%;
+
+}
+
+.contact {
+
+	border-top: 1px solid black;
+	padding-top: 40px;
+	margin-top: 50px;
+
+}
+
+.pg1footer {
+
+	padding-top: 50px;
+	height: 200px;
```

Comment:
Why have a set height? You don't necessarily need the extra space at the bottom for anything from what I'm able to tell.